animatch/LogIn/LogInView.swift

```diff
@@ -0,0 +1,134 @@
+//
+//  LogInView.swift
+//  animatch
+//
+//  Created by Ксения Панкратова on 23.11.2024.
+
+import SwiftUI
+
+struct LogInView: View {
+    @State private var inputPassword = ""
+    @State private var inputEmail = ""
+
+    var body: some View {
+        VStack {
+            ZStack {
+                backgroundLineGradient
+
+                VStack(spacing: 0) {
+                    Logo
+                        .padding(.bottom, 66)
+                    formsContainer
+                    buttonsContainer
+                    Spacer()
+                    footer
+                }
+            }
+        }
+        .ignoresSafeArea()
+    }
+}
+
+// MARK: - UI Subviews
+
+private extension LogInView {
+
+    var backgroundLineGradient: some View {
+        LinearGradient(
+            gradient: Gradient(
+                colors: [
+                    .editProfLightGrad,
+                    .editProfDarkGrad
+                ]
+            ),
+            startPoint: .top,
+            endPoint: .center
+        )
+        .ignoresSafeArea()
+    }
+
+    var formsContainer: some View {
+        VStack(spacing: 0) {
+            Text(Constants.formsContainerTitle)
+                .foregroundStyle(Color.editProfWhite)
+                .font(Font.custom("Roboto", size: 32))
+                .padding(.bottom, 53)
+
+            NGGTextField(title: Constants.nameTextFieldPlaceholder, text: $inputEmail)
+            NGGSecureField(Constants.passwordTextFieldPlaceholder, text: $inputPassword)
+        }
+    }
+
+    var Logo: some View {
```

Comment:
В Swift функции и переменный надо писать с маленькой. И лучше ещё указать, что это View
```suggestion
    var logoView: some View {
```